source/dom.js

```diff
@@ -1,4 +1,10 @@
-const div = document.createElement('div');
+import { isBrowserEnv } from './utils';
+
+const fakeWindow = {};
+
+export const global = isBrowserEnv ? window : fakeWindow;
+
+const div = isBrowserEnv ? document.createElement('div') : {};
```

Comment:
I’m absolutely OK with moving `div` into the method; I was just saying that I don’t think it’s required (because it was not @effrenus who put `div` into the global space).